style.css

```diff
@@ -338,3 +366,101 @@ html {
   color: #fff;
   transition: 0.3s;
 }
+
+@media screen and (min-width: 768px) {
+  .container {
+    margin: auto 10%;
+  }
+
+  /* NAVBAR SECTION  */
+  .navbar {
+    padding: 3.2rem 8.8rem;
+  }
+
+  .nav-btn {
+    display: none;
+  }
+
+  .nav-list {
+    list-style: none;
+    display: flex;
+    gap: 3.2rem;
+  }
+
+  .nav-list a:link,
+  .nav-list a:visited {
+    text-decoration: none;
+    color: #344563;
+    font-size: 1.5rem;
+    font-weight: 500;
+  }
+
+  .nav-list a:hover,
+  .nav-list a:active {
+    border-bottom: 0.3rem solid #6070ff;
+  }
+
+  /* HEADLINE SECTION  */
+  .headline {
+    background: url(images/Desktop/Headline-bg.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: auto auto;
+    margin: 6.4rem 4.8rem;
+  }
+
+  /* PROJECT SECTION  */
+
+  .projects {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 4.8rem;
+    padding: 2.8rem;
+  }
+
+  .project-img {
+    align-self: center;
+  }
+
+  .projects:nth-child(even) > .project-img {
+    order: 2;
+  }
+
+  .btn {
+    justify-self: left;
+  }
+
+  /* ABOUT SECTION  */
+  .languages-elements {
+    flex-direction: row;
+  }
+
+  .languages-elements li {
+    flex-direction: column;
+    align-items: flex-start;
+    width: 12rem;
+    gap: 1.8rem;
+  }
+
+  /* CONTACT SECTION  */
+  .contact {
+    background-image: url(images/Desktop/contact-form-background-shapes.svg);
+  }
+
+  .contact-desc {
+    margin: auto 20%;
+  }
+
+  /* ABOUT SECTION  */
+  .about {
+    grid-template-columns: 1fr 1.1fr;
+  }
```

Comment:
- [ ] According to the design, the left side of the about section covers about 40% of the parent width while the right side occupies 60%. I think the following will get the job done. Please update it accordingly. 😎️
```
  grid-template-columns: 4fr 6fr;
```